src/rest/Facet/FacetValueState.ts

```diff
@@ -1,4 +1,16 @@
+/**
+ * The allowed states of a facet value in a Search API facet
```

Comment:
For consistency with the component name, should it be `... in a Dynamic Facet`? Could `Search API facet` be confusing?

Reply:
Again, the line break makes things slightly confusing. The whole sentence is:

> The allowed states of a facet value in a Search API facet request or response.

I'll remove those line breaks. They seemed like a good idea to keep a reasonable width, but I realize they're making the sentences harder to read which kind of defeats the purpose :)